.devcontainer/devcontainer.json

```diff
@@ -1,39 +1,31 @@
+// For more details, see https://aka.ms/devcontainer.json.
 {
-  "name": "Mastodon",
-  "dockerComposeFile": "docker-compose.yml",
-  "service": "app",
-  "workspaceFolder": "/mastodon",
+		"name": "Mastodon",
+		"dockerComposeFile": "docker-compose.yml",
+		"service": "app",
```

Comment:
Why change the indentation throughout this file? The new indentation doesn't match our `.editorconfig` (https://github.com/mastodon/mastodon/blob/main/.editorconfig) and makes the diff harder to review.

Reply:
Reverted the tab indentation changes 👍